src/sysext/sysext.c

```diff
@@ -962,8 +962,13 @@ static int maybe_import_ignored_mutable_directory(OverlayFSPaths *op) {
                 return log_oom();
 
         r = chase(path, arg_root, CHASE_PREFIX_ROOT, &resolved_path, NULL);
-        if (r < 0 && r != -ENOENT)
+        if (r < 0) {
+                if (r == -ENOENT) {
+                        log_debug("Mutable directory for %s does not exist, not importing", op->hierarchy);
+                        return 0;
+                }
                 return log_error_errno(r, "Failed to resolve mutable directory '%s': %m", path);
+        }
```

Comment:
i would format that as:

```c
if (r == -ENOENT) {
        …
}
if (r < 0)
        …
```

but doesn't matter.

(shallow nesting is best nesting)

Reply:
Fixed anyway.